modules/adding-a-model-server-to-your-data-science-project.adoc

```diff
@@ -47,23 +47,23 @@ The *Add model server* dialog opens.
 .. In the *CPU limit* field, specify the maximum number of CPUs to use with your model server. Use the list beside this field to specify the value in cores or millicores.
 .. In the *Memory requested* field, specify the requested memory for the model server in gibibytes (Gi).
 .. In the *Memory limit* field, specify the maximum memory limit for the model server in gibibytes (Gi).
-. Optional: In the *Model server GPUs* field, specify a number of GPUs to use with your model server.
 +
 [IMPORTANT]
 ====
 {productname-short} includes two versions of the OpenVINO Model Server (OVMS) runtime by default; a version that supports GPUs and one that does not. To use GPUs, from the *Serving runtime* list, you must select the version whose display name includes `Supports GPUs`.
 
 If you are using a _custom_ model-serving runtime with your model server, you must ensure that your custom runtime supports GPUs and is appropriately configured to use them.
 ====
-
+. Optional: From the *Accelerator* field, select an accelerator from the list. 
+. Optional: If you selected an accelerator in the preceding step, from the *Number of accelerators* list, select the number of accelerators to use.  
```

Comment:
Could trim this down to
'Optional: If you selected an accelerator in the preceding step, from the Number of accelerators list, select a value.'

Reply:
Changed.